CHANGELOG.md

```diff
@@ -0,0 +1,36 @@
+# Changelog
+
+All notable changes to this project will be documented in this file. See [standard-version](https://github.com/conventional-changelog/standard-version) for commit guidelines.
+
+## [2.0.0](https://github.com/Tomut0/Dynmap-SimpleClans/compare/v1.4.2...v2.0.0) (2022-06-05)
```

Comment:
Wrong repository (in the entire file)

Reply:
No idea how can I solve it. 🤔
Re-run standard-version, probably.

Reply:
standard-version may have fetched it from your origin repository

Reply:
yeah, I can't run it from your repository, can I? 🤔

Reply:
you can set _this_ repository as origin.